templates/projects/hello-world/data.js

```diff
@@ -22,10 +22,39 @@ template.server = {
 
   modelConfigs: [
     {
+      name: 'User',
+      dataSource: 'db',
+    },
+    {
+      name: 'AccessToken',
+      dataSource: 'db',
+      public: false,
+    },
+    {
+      name: 'ACL',
+      dataSource: 'db',
+      public: false,
+    },
+    {
+      name: 'RoleMapping',
+      dataSource: 'db',
+      public: false,
+    },
+    {
+      name: 'Role',
+      dataSource: 'db',
+      public: false,
+    },
+    {
       name: 'Message',
       dataSource: null,
     },
   ],
 
-  datasources: [],
+  datasources: [ 
+    {
+      name: 'db',
+      connector: 'memory',
+    },
+  ],
```

Comment:
Ah, github diff did not make it clear to me that there alredy is a unit-test for ALCs in hello world.